File: MagniWebAppImportExport/ClientApp/src/models/Importation/NovaColunaModel.ts
export interface NovaColunaModel {
    linhas: any[],
    novaColuna: {
        nome: string;
        coluna: NovaColuna[]
        formato: string;
    }
}

export interface NovaColuna {
    key: number;
    formato: string;
    valor: string;
}

File: MagniWebAppImportExport/Models/Coluna/NovaColunaModel.cs
namespace MagniWebAppImportExport
{
    public class NovaColunaModel
    {
        public object[]? linhas { get; set; }
        public NovaColunaObj? NovaColuna { get; set; }
    }

    public class NovaColunaObj
    {
        public string? Nome { get; set; }
        public string? Formato { get; set; }
        public NovaColuna[]? Coluna { get; set; }
    }

    public class NovaColuna
    {
        public string? Valor { get; set; }
    }
}